src/main/java/me/zeroeightsix/kami/util/Coord.java

```diff
@@ -0,0 +1,38 @@
+package me.zeroeightsix.kami.util;
+
+import net.minecraft.util.math.BlockPos;
+
+/**
+ * @author wnuke
+ * Created by wnuke on 17/04/20
+ */
+
+public class Coord {
+    public int x;
+    public int y;
+    public int z;
+    public boolean chunk;
+    public String name;
+    public String time;
+    public String date;
+
+    public BlockPos getPos() {
+        return new BlockPos(this.x, this.y, this.z);
+    }
+
+    public boolean isChunkCoord() {
```

Comment:
Why are you differentiating between chunk and non chunk? Why not just store all coordinates in their regular format?

Reply:
Because someone suggested to have the option to save the coordinates in chunk format.

Reply:
Tell them all they have to do is divide it by 16 lmao

Reply:
Hmm, yeah, I removed that.